python/slsdet/detector.py

```diff
@@ -3498,10 +3498,57 @@ def clkfreq(self):
 
     def readout(self):
         """
-        Mythen3] Starts detector readout. Status changes to TRANSMITTING and automatically returns to idle at the end of readout.
+        [Mythen3] Starts detector readout. Status changes to TRANSMITTING and automatically returns to idle at the end of readout.
         """
         self.startDetectorReadout()
     
+    @property
+    @element
+    def polarity(self):
```

Comment:
Polarity enum is missing from python. You can get it by running python generate_enums.py from the same folder as the function script.

Reply:
done